pubspec.yaml

```diff
@@ -27,6 +27,7 @@ dependencies:
     path: ./common
   realm_generator: 
     path: ./generator
+  tar: 0.5.0
```

Comment:
```suggestion
  tar: ^0.5.0
```

Reply:
this is on purpose like that. we need exactly this version